_inc/client/plans/single-product-backup.jsx

```diff
@@ -158,6 +159,23 @@ class SingleProductBackupBody extends React.Component {
 		this.props.setSelectedBackupType( event.target.value );
 	};
 
+	handleUpgradeButtonClick = selectedBackupType => () => {
+		analytics.tracks.recordJetpackClick( {
+			target: `upgrade-${ selectedBackupType }`,
+			type: 'upgrade',
+			product: selectedBackupType,
+			page: 'Plans',
```

Comment:
Can we get this to come from the actual page that the user is on. 
Since this would also appear on the plans-prompt page.